test/unit/cli/test_cli.py

```diff
@@ -0,0 +1,145 @@
+import getpass
+import itertools
+import os
+from inspect import cleandoc
+from pathlib import Path
+from unittest.mock import Mock
+
+import click
+import pytest
+from click.testing import CliRunner
+
+from exasol.nb_connector.cli import commands
+from exasol.nb_connector.cli.processing.backend_selector import BackendSelector
+from exasol.nb_connector.secret_store import Secrets
+
+
+def assert_error(result: click.testing.Result, message: str):
+    assert result.exit_code != 0
+    assert message in result.output
+
+
+@pytest.mark.parametrize(
+    "args",
+    [
+        (commands.configure, ["onprem"]),
+        (commands.configure, ["saas"]),
+        (commands.configure, ["docker-db"]),
+        (commands.show, []),
+    ],
+)
+def test_missing_scs_file(args):
+    result = CliRunner().invoke(*args)
+    assert_error(result, "Error: Missing argument 'SCS_FILE'")
+
+
+@pytest.fixture
+def scs_file(tmp_path) -> Path:
+    return tmp_path / "sample.sqlite"
+
+
+@pytest.fixture
+def sample_scs(scs_file, monkeypatch) -> Secrets:
```

Comment:
Why don't you use the `secrets` fixture in the confest.py?

Reply:
Great idea. I was too much focussed on the task at hand rather than perceiving existing fixtures.
Please PR updated by next push.